lib/msal-browser/docs/errors.md

```diff
@@ -114,8 +114,106 @@ msalInstance.handleRedirectPromise()
+
+❌ The following example will throw this error because the `acquireTokenPopup` in the **catch** block does not check if there is another interaction taking place at the moment:
+
+```javascript
+async function myAcquireToken(request) {
+    const msalInstance = getMsalInstance(); // get the msal application instance
+
+    const tokenRequest = {
+        account: msalInstance.getActiveAccount() || null;
+        ...request
+    };
+
+    let tokenResponse;
+
+    try {
+        // attempt silent acquisition first
+        tokenResponse = await msalInstance.acquireTokenSilent(tokenRequest);
+    } catch (error) {
+        if (error instanceof InteractionRequiredAuthError) {
+            try {
+                tokenResponse = await msalInstance.acquireTokenPopup(tokenRequest);
+            } catch (err) {
+                console.log(err);
+                // handle other errors
+            }
+        }
+
+        console.log(error);
+        // handle other errors
+    }
+
+    return tokenResponse;
+}
+
+const request = {
+    scopes: ["User.Read"]
+};
+
+myAcquireToken(request);
+myAcquireToken(request);
+```
+
+✔️ To resolve, you should wait for the interaction status to be `None` before calling any other interactive API:
+
+```javascript
+async function myAcquireToken(request) {
+    const msalInstance = getMsalInstance(); // get the msal application instance
+
+    const tokenRequest = {
+        account: msalInstance.getActiveAccount() || null;
+        ...request
+    };
+
+    let tokenResponse;
+
+    try {
+        // attempt silent acquisition first
+        tokenResponse = await msalInstance.acquireTokenSilent(tokenRequest);
+    } catch (error) {
+        if (error instanceof InteractionRequiredAuthError) {
+            try {
+                // check for any interactions
+                if (myGlobalState.getInteractionStatus() !== InteractionStatus.None) {
+
+                    /**
+                     * "myWaitFor" method polls the interaction status via getInteractionStatus() from
+                     * the application state and resolves when it's equal to "None".
+                     */
+                    await myWaitFor(() => myGlobalState.getInteractionStatus() === InteractionStatus.None);
+
+                    // wait is over, call myAcquireToken again to re-try acquireTokenSilent
+                    return (await myAcquireToken(tokenRequest));
+                } else {
+                    // no interaction, invoke popup flow
+                    tokenResponse = await msalInstance.acquireTokenPopup(tokenRequest);
+                }
+            } catch (err) {
+                console.log(err);
+                // handle other errors
+            }
+        }
+
+        console.log(error);
+        // handle other errors
+    }
+
+    return tokenResponse;
+}
+
+const request = {
+    scopes: ["User.Read"]
+};
+
+myAcquireToken(request);
+myAcquireToken(request);
```

Comment:
Feel free to refactor to clean this up if needed:
```suggestion
myAcquireToken(request).catch((e) => {
  /**
                     * "myWaitFor" method polls the interaction status via getInteractionStatus() from
                     * the application state and resolves when it's equal to "None".
                     */
                    await myWaitFor(() => myGlobalState.getInteractionStatus() === InteractionStatus.None);
                    // wait is over, call myAcquireToken again to re-try acquireTokenSilent
                    return (await myAcquireToken(tokenRequest));
});
myAcquireToken(request).catch((e) => {
  /**
                     * "myWaitFor" method polls the interaction status via getInteractionStatus() from
                     * the application state and resolves when it's equal to "None".
                     */
                    await myWaitFor(() => myGlobalState.getInteractionStatus() === InteractionStatus.None);
                    // wait is over, call myAcquireToken again to re-try acquireTokenSilent
                    return (await myAcquireToken(tokenRequest));
});
```

Reply:
done! I've just put it in a function to avoid repetition